src/ctim/schemas/vocabularies.cljc

```diff
@@ -1509,3 +1509,16 @@
 (def-enum-type PromotionMethod promotion-method
   :open? false
   :gen (cs/gen promotion-method))
+
+(def note-class
```

Comment:
if we proxify it, we could make it open and restrict values in the proxy

Reply:
that's a good point

Reply:
done